messages/lightning.preview.app.md

```diff
@@ -29,11 +27,11 @@ Username or alias of the target org. Not required if the `target-org` configurat
 
 # flags.device-type.summary
 
-Type of device to emulate in preview.
+Type of device to emulate in the preview.
 
 # flags.device-id.summary
 
-For mobile virtual devices, specify the device ID to preview. If omitted, the first available virtual device will be used.
+ID of the mobile device to emulate the preview when device type is set to `mobile`. The default value is the ID of the first available mobile device.
```

Comment:
This is another instance where I'd like to get away from the language of "emulating", because it implies that the preview itself is the emulation, and that's a somewhat incongruent description of what we're doing. I'll make a suggestion that moves away from the language of emulation, but feel free to wordsmith this verbiage further as you will.

```suggestion
ID of the virtual mobile device to be used for previewing, when the device type is set to `mobile`. The default value is the ID of the first available mobile device.
```